script.js

```diff
@@ -191,3 +191,16 @@ const generateProjects = () => {
 window.onload = () => {
   generateProjects();
 };
+
+const form = document.querySelector('#form');
+const errorMail = document.querySelector('#mail-error');
+const email = document.querySelector('#email');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  errorMail.innerText = '';
+  if (/[a-z]/.test(email.value) && /[A-Z]/.test(email.value)) {
+    errorMail.innerText = 'Please a provide valid lowercase email address';
+  } else {
+    form.submit();
+  }
+});
```

Comment:
- [ ] Every requirement and form validation condition has been implemented correctly, but kindly note that if your form is ready for submission and you click on the button, it does not work, and throws that error, kindly address this issue. 
  
 ![image](https://github.com/SnakyMz/Portfolio/assets/102612396/2b776835-1915-4bc0-bd28-56310c9dc9f3)

Reply:
Please any advice why this could be happening?

Reply:
Hello @SnakyMz,
When you give your `form button` and `id` of "submit", you kind of like override the submit() function on this code, so simply put, change the `id` name of your `form button`.

Reply:
Thank you very much. I have learned something new thanks to you. Ill make changes rightaway

Reply:
I wish you all the best, you have what it takes.

Reply:
Thank you very much. Best of Luck to you too. May you have prosperous life up ahead.